### ISSUE TITLE
Cannot set Keywordize to false

### ISSUE THREAD

**Issue by whamtet** [michaelklishin/monger#108]
By default monger keywordizes clojure maps.  Presumably I can turn this off

```clojure
(require '[monger.collection :as mc])
(let [keywordize? false]
    (mc/find-map-by-id db "collection" "id" {} keywordize?))
```

The output is still keywordized.

**Comment by kronos** [comment]
Should be fixed in #135

**Comment by whamtet** [comment]
Thanks!
